src/Sulu/Bundle/MediaBundle/Entity/CollectionRepository.php

```diff
@@ -339,6 +342,8 @@ private function getIdsQuery(
             }
         }
 
+        $queryBuilder->addOrderBy('collection.id', 'ASC');
```

Comment:
1. It is, as the order is indeterministic in postgres (or at leas different form the one in Mysql)
2. If you sort by a property for which to collections have the same value, the same issue occurs. Within the equivalence class of one property the order of the others is not determined.